rust/Cargo.toml

```diff
@@ -37,12 +37,16 @@ fr32 = { version = "~5.0", default-features = false }
 fvm3 = { package = "fvm", version = "3.0.0-alpha.12", default-features = false, features = ["f4-as-account"] }
 fvm3_shared = { package = "fvm_shared", version = "3.0.0-alpha.13" }
 fvm3_ipld_encoding = { package = "fvm_ipld_encoding", version = "0.3.0" }
+fvm3_ipld_blockstore = {package = "fvm_ipld_blockstore", version = "0.1.2" }
 fvm2 = { package = "fvm", version = "2.1.0", default-features = false }
 fvm2_shared = { package = "fvm_shared", version = "2.0.0" }
 fvm2_ipld_encoding = { package = "fvm_ipld_encoding", version = "0.2.2" }
-fvm_ipld_blockstore = "0.1.1"
+fvm2_ipld_blockstore = { package = "fvm_ipld_blockstore", version = "0.1.1" }
 num-traits = "0.2.14"
-cid = { version = "0.8.5", features = ["serde-codec"] }
+fvm3_cid = { package = "cid", version = "0.9.0", features = ["serde-codec"] }
+fvm2_cid = { package = "cid", version = "0.8.6", features = ["serde-codec"] }
+fvm3_multihash = { package = "multihash", version = "0.17.0"}
+fvm2_multihash = { package = "multihash", version = "0.16.3"}
```

Comment:
Will do, we will need to create new version of fvm3 with the new CID and multihash crates.  One annoying thing with this patch is once applied it will not allow FVM2 and FMV3 to be on the same version of CID and multihash (aliasing crates to same crate version error).